build.sh

```diff
@@ -15,6 +15,9 @@ rm -rf pkg/*
 rm -rf $GOPATH/pkg/*
 mkdir -p bin/
 
+# Fix for build failing due to missing go-vnc
+(cd $GOPATH/src/github.com/mitchellh && if cd go-vnc; then git pull; else git clone https://github.com/mitchellh/go-vnc.git; fi)
```

Comment:
Yes, I followed exactly:
1) https://github.com/mitchellh/packer/blob/master/CONTRIBUTING.md#setting-up-go-to-work-on-packer
2) https://github.com/xenserver/packer-builder-xenserver/blob/master/README.md#compile-the-plugin

Perhaps I should change this to something like `go get go-vnc`, but either way, `go-vnc` is never downloaded in either process. It's happened to both me and another guy: https://github.com/xenserver/packer-builder-xenserver/issues/45

**ASIDE:**
I've been working with packer-builder-xenserver for Windows and I've noticed the following problems:
1) WinRM timeout is defaulted to 0s (expecting 20 minutes)
2) ssh_username is still required (should be winrm_username)
3) plugin does not correctly get winrm host ip
4) plugin does not correctly set winrm port, and specifying winrm_port in the config is ignored

I've made some ['dirty patches' for 3 and 4](https://github.com/dongyuzheng/packer-builder-xenserver/tree/winrm_fix) to make it work in the meanwhile, but please take a look :)